src/pytest/test_issues.py

```diff
@@ -13,7 +13,7 @@ def test_1304(env):
 
 def test_1414(env):
   env.skipOnCluster()
-  env.expect('FT.CREATE idx SCHEMA txt1 TEXT').equal('OK')
+  env.expect('FT.CREATE idx SCHEMA foo TEXT').equal('OK')
```

Comment:
w/o any field that matches the scehma, the result would be `[0L]`.
Before the fix, searching for `*` would have returned all the docs in the index even if none of their terms was indexed.

IMO, this is more correct and will save some space in the doctable.

Reply:
We can change it back to txt1 now no?